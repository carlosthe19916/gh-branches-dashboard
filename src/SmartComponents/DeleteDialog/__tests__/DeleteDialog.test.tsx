import React from "react";
import { shallow } from "enzyme";
import { DeleteDialogBase, DeleteDialogBaseProps } from "../DeleteDialog";

it("Test snapshot", () => {
  const props: DeleteDialogBaseProps = {
    name: "myFolder",
    type: "Folder",
    isOpen: false,
    isProcessing: false,
    isError: false,
    onDelete: jest.fn(),
    onCancel: jest.fn()
  };

  const wrapper = shallow(<DeleteDialogBase {...props} />);
  expect(wrapper).toMatchSnapshot();
});
