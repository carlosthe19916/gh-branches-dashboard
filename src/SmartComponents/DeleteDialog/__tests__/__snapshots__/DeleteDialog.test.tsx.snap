// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
<Modal
  actions={
    Array [
      <Unknown
        isDisabled={false}
        onClick={[MockFunction]}
        variant="danger"
      >
        Delete 
        Folder
      </Unknown>,
      <Unknown
        isDisabled={false}
        onClick={[Function]}
        variant="link"
      >
        Cancel
      </Unknown>,
    ]
  }
  appendTo={<body />}
  ariaDescribedById=""
  className=""
  hideTitle={false}
  isFooterLeftAligned={false}
  isLarge={false}
  isOpen={false}
  isSmall={true}
  onClose={[Function]}
  showClose={true}
  title="Delete myFolder?"
>
  Are you sure you want to delete this Folder? This action will remove any data related to this Folder permanently.
</Modal>
`;
